hypha/apply/funds/models/submissions.py

```diff
@@ -634,6 +635,12 @@ def save(self, *args, update_fields=list(), skip_custom=False, **kwargs):
             self.draft_revision = first_revision
             self.save()
 
+    @property
+    def has_all_reviewer_roles_assigned(self):
+        all_role_ids = set(ReviewerRole.objects.values_list('id', flat=True))
+        assigned_role_ids = set(self.assigned.with_roles().values_list('role__id', flat=True))
```

Comment:
`assignedReviewers` is unique together with the role so you should be able to just `count()` roles and assignees here.
 https://github.com/OpenTechFund/hypha/blob/6ea1d55451f5f1d2231605553c0d79833dd62fb0/hypha/apply/funds/models/submissions.py#L962-L963

Reply:
yeah, that's better. I've updated that